packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -49,21 +49,23 @@ export class ComponentPageBase extends React.PureComponent<IComponentPageProps>
   public render() {
     const { componentName, className, otherSections, styles, theme } = this.props;
 
+    const showOnlyExamples = location && location.hash.indexOf('docsExample=true') > -1;
```

Comment:
I'm not sure this makes sense--how would you put a query parameter into the hash (along with the route)?

Reply:
I am planning to automatically generate a markdown file with an iframe pointing to this url along with the query parameter. So we are controlling the generation of those iframes and the `src` attribute. That is why I need a dev link to test it. I will try your suggestion to try and pass the ssr-tests and see how it goes.